content/docs/concepts/components/stack.md

```diff
@@ -72,87 +72,11 @@ $ pack build example/app
 > a given builder, among other useful information. The order of the run images in the output denotes the order in
 > which they will be matched during `build`.
 
-## Creating custom stacks
-
-To create a custom stack, simply create customized build and run images containing the following information:
-
-### Labels
-
-| Name | Description | Format
-|------|-------------|--------
-| `io.buildpacks.stack.id` | Identifier for the stack | String
-
-### Environment Variables
-
-| Name | Description |
-|------|-------------|
-| `CNB_STACK_ID` | Identifier for the stack |
-| `CNB_USER_ID`  | UID of the user specified in the image |
-| `CNB_GROUP_ID` | GID of the user specified in the image |
-<p class="spacer"></p>
-
-> **NOTE:** The **stack identifier** implies compatibility with other stacks of that same identifier. For instance, a custom stack may use
-> `io.buildpacks.stacks.bionic` as its identifier so long as it will work with buildpacks that declare compatibility with the
-> `io.buildpacks.stacks.bionic` stack.
-
-## Mixins
-
-Mixins provide a way to document OS-level dependencies that a stack provides to buildpacks. Mixins can be provided at build-time
-(name prefixed with `build:`), run-time (name prefixed with `run:`), or both (name unprefixed).
-
-### Declaring provided mixins
-
-When declaring provided mixins, both the build and run image of a stack must contain the following label:
-
-| Name | Description | Format
-|------|-------------|--------
-| `io.buildpacks.stack.mixins` | List of provided mixins | JSON string array
-
-\
-The following rules apply for mixin declarations:
-
- - `build:`-prefixed mixins may not be declared on a run image
- - `run:`-prefixed mixins may not be declared on a build image
- - Unprefixed mixins must be declared on both stack images
-
-#### Example
-
-_Build image:_
-```json
-io.buildpacks.stack.mixins: ["build:git", "wget"]
-```
-
-_Run image:_
-```json
-io.buildpacks.stack.mixins: ["run:imagemagick", "wget"]
-```
-
-### Declaring required mixins
-
-A buildpack must list any required mixins in the `stacks` section of its `buildpack.toml` file.
-
-When validating whether the buildpack's mixins are satisfied by a stack, the following rules apply:
-
-- `build:`-prefixed mixins must be provided by stack's build image
-- `run:`-prefixed mixins must be provided by stack's run image
-- Unprefixed mixins must be provided by both stack images
-
-#### Example
-
-```toml
-[[stacks]]
-id = "io.buildpacks.stacks.bionic"
-mixins = ["build:git", "run:imagemagick", "wget"]
-```
-
 ## Resources
 
-For sample stacks, see our [samples][samples] repo.
-
-For technical details on stacks, see the [platform specification for stacks][stack-spec].
+To learn how to create your own stack, see our [Operator's Guide][operator-guide].
```

Comment:
Moved it to the operator's guide as I felt that would be an appropriate place for something like a stack. We can also move it to a different section entirely if we think that's better.